site/pages/api/admin/update-email-preference.api.ts

```diff
@@ -16,8 +16,6 @@ const updateEmailPreference = async (req: NextApiRequest, res: NextApiResponse)
 
         if (!session) {
             throw new Error("No session found");
-        } else if (!session.isOrgAdmin) {
-            throw new Error("Invalid user accessing the page");
```

Comment:
Emily said For any preferences e.g. email notifications or data sources, we only present these choices to organisation admin accounts currently - so I think we should still have this check

Reply:
Checked with Emily and she's happy to leave code as is so we can set email preferences for street manager for org admins, publishers and staff.